skorch/tests/test_helper.py

```diff
@@ -708,3 +708,47 @@ def test_describe_signature_other_dtypes(self, transformer_cls, df):
             'col_cats': {"dtype": torch.int32, "input_units": 2},
         }
         assert result == expected
+
+
+class TestAccelerate:
+    @pytest.fixture(scope='module')
+    def data(self, classifier_data):
+        return classifier_data
+
+    @pytest.fixture(scope='module')
+    def module_cls(self, classifier_module):
+        return classifier_module
+
+    @pytest.fixture(scope='module')
+    def net_cls(self):
+        from skorch import NeuralNetClassifier
+        from skorch.helper import AccelerateMixin
+
+        class AcceleratedNet(AccelerateMixin, NeuralNetClassifier):
+            pass
+
+        return AcceleratedNet
+
+    @pytest.fixture
+    def accelerator(self):
+        pytest.importorskip('accelerate')
+
+        from accelerate import Accelerator
+
+        accelerator = Accelerator(fp16=True)
+        return accelerator
+
+    @pytest.fixture
+    def net(self, net_cls, module_cls, accelerator):
+        return net_cls(
```

Comment:
As a sanity check, it would be good to check if the `NeuralNet` can be pickled and unpickled with an accelerator.

Reply:
Good point.